docs/computer/Acoustics.md

```diff
@@ -63,21 +63,21 @@ Key features:
 3.Inbuilt IMU that can be used further for movement of the vehicle.
 4.WiFi connectivity.
 
-![NI myRIO-1900 Hardware Block Diagram](https://github.com/auvzhcet/Documentation/blob/jawad-patch-1/docs/computer/static/myRIO_hardware_pinout.png)
+![NI myRIO-1900 Hardware Block Diagram](static/myRIO_hardware_pinout.png)
 
-![Primary/Secondary Signals on MXP Connectors A and B](https://github.com/auvzhcet/Documentation/blob/jawad-patch-1/docs/computer/static/myRio_ports.png)
-_Source: [NI MyRIO User Guide and Specification](https://github.com/auvzhcet/Documentation/blob/jawad-patch-1/docs/computer/static/NI_MyRIO_User_Guide_and_Specification.pdf)_
+![Primary/Secondary Signals on MXP Connectors A and B](static/myRio_ports.png)
+_Source: [NI MyRIO User Guide and Specification](static/NI_MyRIO_User_Guide_and_Specification.pdf)_
```

Comment:
Is this working fine locally?

Reply:
If yes! then please change other pdf links also. Else, let other pdf links to be like that github link.